packages/library-legacy/src/programs/stake.ts

```diff
@@ -776,7 +777,8 @@ export class StakeProgram {
     if (custodianPubkey) {
       keys.push({
         pubkey: custodianPubkey,
-        isSigner: false,
+        isSigner: stakeAuthorizationType.index
+            === StakeAuthorizationLayout.Withdrawer.index,
```

Comment:
This one looks like it's a signer whenever provided:
<https://github.com/solana-labs/solana/blob/fca44b78a681ac1abf7878079e8a7a319c31b23f/sdk/program/src/stake/instruction.rs#L618-L620>

Reply:
> But as the `rs` code sets it to `true` all the time then you would consider to have that consistent and set it up to `true` always, would that be so?

Yeah, ideally if the crate is building an instruction one way, we want to exactly mimic that in JS unless there's a bug report associated with it.

Reply:
I see, thanks for clarification. I updated the PR to define the signer being `true` on all three places.